upload-api/test/service/delegations.test.js

```diff
@@ -21,12 +23,17 @@ test.before(async (t) => {
 testDelegationsStorageVariant(
   async (/** @type {any} */ t) => {
     const { dynamo, s3 } = t.context
-    const bucketName = await createBucket(s3)
+    const delegationsBucketName = await createBucket(s3)
+    const invocationsBucketName = await createBucket(s3)
+    const workflowBucketName = await createBucket(s3)
+
     
     return useDelegationsTable(
       dynamo,
       await createTable(dynamo, delegationTableProps),
-      useDelegationsStore(s3, bucketName)
+      useDelegationsStore(s3, delegationsBucketName),
+      useInvocationStore(s3, invocationsBucketName),
+      useWorkflowStore(s3, workflowBucketName)
     )
```

Comment:
I think that we should be able to test this flow. Would expect something like this https://github.com/web3-storage/w3infra/blob/main/upload-api/test/ucan-invocation.test.js#L140-L157 (using access api invocations for the delegations, meaning it would be stored in the invocations bucket and workflow bucket)

Reply:
yep agree! I'm going to get this merged into the main PR and add a test there